### PR TITLE
Added Ukrainian translation to built-in FAQ page

### PR DESCRIPTION
**Pull request by strump**
Added Ukrainian translation. Tested in browser. 

Some places in the FAQ are outdated. E.g. Button "Routes" is available only in iOS. Taxi routing is removed. Phrase "You can add up to 3 intermediate points to a route." is outdated. The same regarding "On Android, please make sure that you have granted network access for Organic Maps and system Download Manager (Download Provider)."

### CODE REVIEW

**Comment by biodranik** [comment]
@strump thanks! Can you please make a new issue and list found problems that should be updated in different languages? Or even better, replace the mentioned strings using DeepL translate.py script, and create a PR with the proposed fix, we'll tag our translators to review and fix.

Later these FAQ items will go to the website too.

File: data/faq.html [hunk 1]
```diff
@@ -124,6 +124,7 @@ <h2 lang="pt">Perguntas frequentes</h2>
       <h2 lang="pt-BR">Perguntas frequentes</h2>
       <h2 lang="pl">FAQ</h2>
       <h2 lang="tr">SSS</h2>
+      <h2 lang="uk">ЧаПи</h2>
```
Comment (biodranik) [inline data/faq.html:127]: I never saw this ) Maybe use the full version?

File: data/faq.html [hunk 2]
```diff
@@ -2626,7 +2864,7 @@ <h3 lang="tr">Sesli yönlendirmeyi duyamıyorum</h3>
           Arabic, Chinese (Traditional and Simplified), Czech, Danish, Dutch,
           Finnish, French, German, Greek, Hindi, Hungarian, Indonesian, Italian,
           Japanese, Korean, Polish, Portuguese, Romanian, Russian, Slovak,
-          Spanish, Swedish, Thai, Turkish.
+          Spanish, Swedish, Thai, Ukrainian, Turkish.
```
Comment (biodranik) [inline data/faq.html:2867]: Should 26 above be fixed? Is there an official list somewhere? Did you count it yourself? :)

File: data/faq.html [hunk 3]
```diff
@@ -3474,5 +3820,5 @@ <h3 lang="tr">Yer imleri nasıl içe aktarılır</h3>
     <a href="#" lang="pt-BR">Ir para o início</a>
     <a href="#" lang="pl">Idź na górę</a>
     <a href="#" lang="tr">Başa Git</a>
+    <a href="#" lang="uk">До початку</a>
   </body>
-</html>
```
Comment (biodranik) [inline data/faq.html:3478]: ?

File: data/faq.html (hunk 2 above)
Comment (strump) [reply data/faq.html]: Removed Ukrainian. Will change list of languages in next PR.

File: data/faq.html (hunk 3 above)
Comment (strump) [reply data/faq.html:3478]: Reverted missing tag.

File: data/faq.html (hunk 1 above)
Comment (strump) [reply data/faq.html]: [Wikipedia](https://uk.wikipedia.org/wiki/FAQ#.D0.A3.D0.BA.D1.80.D0.B0.D1.97.D0.BD.D1.81.D1.8C.D0.BA.D1.96_.D0.B2.D0.B0.D1.80.D1.96.D0.B0.D0.BD.D1.82.D0.B8_.D0.B0.D0.BA.D1.80.D0.BE.D0.BD.D1.96.D0.BC.D1.83) says that "ЧаПи" is frequently used. But I replaced with full text.

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**